tests/foreman/api/test_classparameters.py

```diff
@@ -584,9 +874,21 @@ def test_positive_create_matcher_puppet_default_value(self):
         puppet default value.
 
         @assert: The matcher has been created successfully.
-
-        @caseautomation: notautomated
         """
+        sc_param = self.sc_params_list.pop()
+        value = gen_string('alpha')
+        sc_param.override = True
+        sc_param.default_value = gen_string('alpha')
+        entities.OverrideValue(
+            smart_class_parameter=sc_param,
+            match='domain=example.com',
+            value=value,
+            use_puppet_default=True,
+        ).create()
+        sc_param = sc_param.read()
+        self.assertEqual(
+            sc_param.override_values[0]['match'], 'domain=example.com')
+        self.assertEqual(sc_param.override_values[0]['value'], value)
 
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1352502

If we have same situation for API, @svtkachenko please create a new defect for that